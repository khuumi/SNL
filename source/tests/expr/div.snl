958.1 / 95