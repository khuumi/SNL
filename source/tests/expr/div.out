Binop (Float 958.1) Div (Int 95)